pkg/client/src/app/pages/identities/components/identity-form.tsx

```diff
@@ -96,6 +148,14 @@ export const IdentityForm: React.FC<IdentityFormProps> = ({
     description: string()
       .trim()
       .max(250, t("validation.maxLength", { length: 250 })),
+    kind: object().shape({
+      value: string().min(1, "value required").required(),
+      toString: object().required(),
+    }),
```

Comment:
This is a detail, `.shape` could be removed as this is equivalent:
```
kind: object().shape({
      value: string().min(1, "value required").required(),
      toString: object().required(),
    }),
```
Actually the whole block can be replaced with:

``` 
const validationSchema = object({
    name: string()
      .trim()
      .required(t("validation.required"))
      .min(3, t("validation.minLength", { length: 3 }))
      .max(120, t("validation.maxLength", { length: 120 })),
    description: string()
      .trim()
      .max(250, t("validation.maxLength", { length: 250 })),
    kind: object().shape({
      value: string().min(1, "value required").required(),
      toString: object().required(),
    }),
    settings: string().when("kind.value", {
      is: "maven",
      then: string().required("Must upload xml settings file"),
    }),
  });

```

Reply:
> This is a detail, `.shape` could be removed as this is equivalent:
> 
> ```
> kind: object().shape({
>       value: string().min(1, "value required").required(),
>       toString: object().required(),
>     }),
> ```
> 
Is this not what is already there? Did you mean to remove "shape()" in your example ablove?
> Actually the whole block can be replaced with:
> 
> ```
> const validationSchema = object({
>     name: string()
>       .trim()
>       .required(t("validation.required"))
>       .min(3, t("validation.minLength", { length: 3 }))
>       .max(120, t("validation.maxLength", { length: 120 })),
>     description: string()
>       .trim()
>       .max(250, t("validation.maxLength", { length: 250 })),
>     kind: object().shape({
>       value: string().min(1, "value required").required(),
>       toString: object().required(),
>     }),
>     settings: string().when("kind.value", {
>       is: "maven",
>       then: string().required("Must upload xml settings file"),
>     }),
>   });
> ```

Reply:
Ah yes, you're right, in first example without shape.